landing/static/js/index.js

```diff
@@ -128,3 +134,26 @@ $('#contact-form').submit(function(event){
         }
     });
 });
+
+function changeLanguage (langCode) {
+    if (langCode === "uk" || langCode === "en") {
+        $("[lang]").each(function () {
+            if ($(this).attr("lang") === langCode)
+                $(this).show();
+            else
+                $(this).hide();
+        });
+    } else {
+        throw new Error("LangCode " + langCode + " not supported");
+        }
+}
+
+$('#select_language').on("change", function() {
+    const language = $(this).val();
+    const userlang = window.localStorage.setItem('lang', language); 
+    changeLanguage(language);
+});
+
+$(document).ready(() => {
+    changeLanguage("uk");
+});
```

Comment:
it remains to check if there is a `lang` in **localStorage** when loading the page, and if so, `changeLanguage` to this value.